src/forms/button/ButtonGroup.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {RadioButton} from "../radio";
+import * as utils from "../../utils";
+import {ReactElement, ReactNode} from "react";
+import {classNames} from "../../utils";
+
+type ButtonGroupProps = {
+    className?: string;
+    children?: React.ReactNode[];
+    style?: any;
+    defaultValue?: any;
+    label?: string;
+    disabled?: boolean;
+    isRadio?: boolean;
+    name: string;
+    onChange?: (evt: React.ChangeEvent<HTMLInputElement>) => void;
+    onClick?: (evt: React.ClickEvent<HTMLInputElement>) => void;
```

Comment:
I don't see any reason why we need onClick property.  We can handle everything through onChange.  The value returned in `evt` of onChange will reflect what button was clicked.  This is also the same logic we followed in RadioButtonGroup.

Reply:
Wen we are using RadioButton inside ButtonGroup, we are actually using 'radio btn' for RadioButton internally. 
When we clicked on RadioButton ('radio btn') , it generate onClick event and not onChange event.

Reply:
So In case of ButtonGroup code, we need to remove the code for onChange as it is not require.